addons/l10n_in/models/res_partner.py

```diff
@@ -16,6 +16,7 @@ class ResPartner(models.Model):
             ('special_economic_zone', 'Special Economic Zone'),
             ('deemed_export', 'Deemed Export'),
         ], string="GST Treatment")
+    l10n_in_pan = fields.Char(string="PAN")
```

Comment:
Why can not this field be put in one of the many other GST fields we already have?

Reply:
PAN and GST number is different so we can't add in GST field